src/core/storage/Logger.h

```diff
@@ -0,0 +1,25 @@
+#ifndef EUGENE_SRC_EUGENE_API_LOGGER_H
+#define EUGENE_SRC_EUGENE_API_LOGGER_H
+
```

Comment:
I am pretty sure this file is at the wrong place. It is not specific to the `storage` module, so it should be moved to `src/core` instead.